### ISSUE TITLE
events_order of RSSAgent when defining an array of urls

### ISSUE THREAD

**Issue by hunhejj** [cantino/huginn#1077]
Hi,
it is possible to define multiple urls for the RSSAgent. In this case the RSS urls are parsed one after one. 

However I want to get a misced output for further processing sorted by publication date. Ok, so I've set the events_order key, but it only effects events created by the SAME url. 

Do you have any solution? What am I doing wrong? 
Thanks,
hunhejj

**Comment by cantino** [comment]
Hey @hunhejj, you're not doing anything wrong, I think that's a bug.  Would you mind seeing if #1079 fixes this for you?

**Comment by hunhejj** [comment]
Thanks for the quick reply @cantino! Honestly I'm not sure how to apply the fix. Where can I access the event source files if I'm using Heroku? Sorry for the stupid question but I'm quite new to GitHub/Linux (but I'm eager to learn!)

**Comment by cantino** [comment]
You'd need to checkout the `rss_agent_sorts_all_events` git branch locally, then push it to Heroku.  It'd be something like:

```shell
git pull
git checkout rss_agent_sorts_all_events
git push heroku rss_agent_sorts_all_events:master
```

**Comment by hunhejj** [comment]
I did the steps you've wrote, but I got the following error message: 
>! [rejected]        rss_agent_sorts_all_events -> master (non-fast-forward)
error: failed to push some refs to 'https://git.heroku.com/(...).git'
hint: Updates were rejected because a pushed branch tip is behind its remote
hint: counterpart. Check out this branch and integrate the remote changes
hint: (e.g. 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I definately need a git crash course :S
